ngraph/test/op_eval/einsum.cpp

```diff
@@ -0,0 +1,237 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <string>
```

Comment:
Please add new tests for Visitor API.
https://github.com/openvinotoolkit/openvino/tree/master/ngraph/test/visitors/op

Reply:
Added.